xbmc/utils/RecentlyAddedJob.cpp

```diff
@@ -258,15 +258,14 @@ bool CRecentlyAddedJob::UpdateMusic()
     home->SetProperty("LatestSong." + value + ".Fanart"  , "");
   }
   
-  i = 0;
```

Comment:
Doesn't the removal of this break https://github.com/xbmc/xbmc/pull/9107/files#diff-59e3b31da92e51a0abb82352afc566d2R282.

If you do not reset i then it's already at NUM_ITEMS and that loop won't run.